src/content/getting-started/prepare-your-provider-account/vmware-vsphere/_index.md

```diff
@@ -3,11 +3,103 @@ title: Prepare your provider account for VMware vSphere
 linkTitle: Prepare your vSphere account
 description: Prepare your VMware vSphere setup to start building your cloud-native developer platform with Giant Swarm.
 weight: 10
-last_review_date: 2024-04-01
+last_review_date: 2024-05-17
 owner:
   - https://github.com/orgs/giantswarm/teams/sig-docs
 user_questions:
   - How do I prepare my VMware vSphere account for the cloud-native developer platform?
   - What do I need to do to prepare my VMware vSphere account for the cloud-native developer platform?
 ---
 
+In order to run the Giant Swarm platform in your VMware vSphere environment, several prerequisites must be satisfied to support Cluster API Provider VMware Cloud Director (CAPVCD).
+
+## Requirements
+
+The solution has some requirements from the vCenter side, and at the same time, the controllers that provision the infrastructure need some permissions again from the vCenter server API.
+
+### VMware vSphere
+
+The minimum version of vSphere required is 6.7 Update 3. It's recommended to run vSphere 7.0 or above. That versions support the Cloud Native Storage (CNS) feature needed for PersistentVolumes (PV) in Kubernetes.
+
+| VMware product | Required version |
+|------|------|
+| VMware virtual hardware | 13 or later |
+| vSphere ESXi hosts | 6.7 Update 3 or later |
+| vCenter host | 6.7 Update 3 or later |
+
+### Resource pool
+
+It's recommended to create one resource pool across the hosts where the workload cluster virtual machines will run. However, in case of inconvenience, it's possible to run the virtual machines in the implicit root resource pool located at the vSphere cluster level.
+
+## Controller permissions
+
+The Cluster API controller that provisions the infrastructure in the vSphere environment needs a role with a set of permissions. To follow the principle of least privilege, it's recommended that a specific user and role be created for the controller.
+
+<!-- Xavier can we provide a text here to create the user too? -->
```

Comment:
@vxav I thought we might also mention user creation if we point to creating an ad-hoc one for CAPV

Reply:
We can point to the documentation but the user could be either local to the vCenter database, or come from an identity source (AD, openLDAP, ...).

https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-authentication/GUID-31F302A6-D622-4FEC-9007-EE3BA1205AEA.html